test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -226,6 +232,43 @@ void ProtoConverter::visit(VarDecl const& _x)
 	m_output << "\n";
 }
 
+void ProtoConverter::visit(EmptyVarDecl const&)
+{
+	m_output << "let x_" << m_numLiveVars++ << "\n";
+	m_numVarsPerScope.top()++;
+}
+
+void ProtoConverter::visit(MultiVarDecl const& _x)
+{
+	size_t funcId = (static_cast<size_t>(_x.func_index()) % m_functionVecMultiReturnValue.size());
+
+	int numInParams = m_functionVecMultiReturnValue.at(funcId).first;
+	int numOutParams = m_functionVecMultiReturnValue.at(funcId).second;
+
+	// Ensure that the chosen function returns at least 2 and at most 4 values
+	yulAssert(
+		((numOutParams >= 2) && (numOutParams <= 4)),
+		"Proto fuzzer: Multi variable declaration calls a function with either too few or too many output params."
+	);
+
+	// We must start variable numbering past the number of live variables at this point in time.
+	// This creates let x_p,..., x_k :=
+	// (k-p)+1 = numOutParams
+	m_output << "let "
```

Comment:
`<<` should always be the last token on the line and the lines below should be indented one tab less.

Reply:
done